R/R.sublime-syntax

```diff
@@ -41,24 +43,61 @@ contexts:
     - include: function-calls
     - include: general-variables
 
+###[ COMMENTS ]################################################################
+
   codesection:
+    - meta_include_prototype: false
```

Comment:
This context is never directly pushed to via `push: codesection`. Thus `meta_include_prototype` is redundant.